aten/src/ATen/native/native_functions.yaml

```diff
@@ -717,15 +717,12 @@
   use_c10_dispatcher: True
   supports_named_tensor: True
   variants: function, method
-  dispatch:
-    CPU: _cosh__cpu
-    CUDA: _cosh__cuda
 
 - func: cosh.out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
   supports_named_tensor: True
   dispatch:
-    CPU: _cosh_out_cpu
-    CUDA: _cosh_out_cuda
+    CPU: _cosh_out
+    CUDA: _cosh_out
```

Comment:
The leading underscores might lead to build errors

Reply:
What would you advise? I believe this is the correct dispatch, and there are many other operators in `native_functions.yaml` that use leading underscores.

Reply:
These two lines point to your newly defined function in UnaryOps.cpp, whose name do not start with `_`. That's why I said that leaving the underscores here will likely to fail the build.